### PR TITLE
Add troubleshooting problem and solution for yarn link

### PR DESCRIPTION
**Pull request by 0Calories**
There's a bug (likely a yarn bug) where you cannot use `yarn link` in your sentry project to link local copies of packages for testing, this is a workaround.

### CODE REVIEW

**Review by AbhiPrasad — APPROVED** [review]

File: src/docs/environment/index.mdx [hunk 1]
```diff
@@ -219,4 +219,14 @@ docker ps
 
 **Problem:** You use an Android emulator with a DSN pointing to localhost, and the events don't show up in your local Sentry instance.
 
-**Solution:** Change `localhost` to `10.0.2.2`. So, for example, change http://d895df97e1cb4a33b4dff8af3e78da09@localhost:8000/2 to http://d895df97e1cb4a33b4dff8af3e78da09@10.0.2.2:8000/2. This is because localhost or `127.0.0.1` refers to the emulator's own loopback interface, not the loopback interface of the host machine. For more information see https://developer.android.com/studio/run/emulator-networking.
\ No newline at end of file
+**Solution:** Change `localhost` to `10.0.2.2`. So, for example, change http://d895df97e1cb4a33b4dff8af3e78da09@localhost:8000/2 to http://d895df97e1cb4a33b4dff8af3e78da09@10.0.2.2:8000/2. This is because localhost or `127.0.0.1` refers to the emulator's own loopback interface, not the loopback interface of the host machine. For more information see https://developer.android.com/studio/run/emulator-networking.
```
Comment (asottile-sentry) [inline src/docs/environment/index.mdx:222]: instead of this we should revert the change that broke this

**Review by asottile-sentry — CHANGES_REQUESTED** [review]
